pkg/test/helm/helm.go

```diff
@@ -35,6 +35,12 @@ func New(kubeConfig string) *Helm {
 	}
 }
 
+// InstallChartWithValues installs the specified chart with its given name to the given namespace
+func (h *Helm) AddRepo() error {
+	_, err := execCommand("helm repo add istio https://istio-release.storage.googleapis.com/charts")
```

Comment:
Could we make this configurable so folks could use this test suite with their own chart repo?

Reply:
Done

Reply:
Wonderful; thank you!